modules/lang-painless/src/main/antlr/SuggestLexer.g4

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+lexer grammar SuggestLexer;
```

Comment:
This is substantily similar to `PainlessLexer.g4`, did you consider factoring out the commonalities and importing them in?

Reply:
I did, but given this is iteration one and there will likely be more changes required, I think it would be beneficial to hold off on any kind of merging until we have a more finalized view.